fbw-a32nx/src/systems/instruments/src/Common/EWDMessages.tsx

```diff
@@ -347,6 +347,8 @@ const EWDMessages = {
     '320005001': '\x1b<2m\x1b4mCONFIG\x1bm PARK BRK ON',
     '320006001': '\x1b<4m\x1b4mBRAKES\x1bm A/SKID N/WS FAULT',
     '320006002': '\x1b<5m MAX BRK PR......1000 PSI',
+    '320015001': '\x1b<2m\x1b4mL/G\x1bm GEAR NOT DOWN',
+    '320015501': '\x1b<2m\x1b4mL/G\x1bm GEAR NOT DOWN',
```

Comment:
Is this expected to be identical?

Reply:
Yes, that is intentional. Good spotting though 😁 .